docs/README.md

```
@@ -29,6 +196,7 @@ Welcome to the Robot SF documentation! This directory contains comprehensive gui
 - [**Pedestrian Metrics Overview**](./ped_metrics/PED_METRICS.md) - Summary of implemented metrics and their purpose
 - [**Metric Analysis**](./ped_metrics/PED_METRICS_ANALYSIS.md) - Overview of metrics used in research and validation
 - [**NPC Pedestrian Design**](./ped_metrics/NPC_PEDESTRIAN.md) - Details on the design and behavior of NPC pedestrians
+ - [**Pedestrian Density Reference**](./ped_metrics/PEDESTRIAN_DENSITY.md) - Units, canonical triad (0.02/0.05/0.08), advisory range, difficulty mapping & test policy
```

markdownlint-cli2 (0.18.1)

Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)